R/symComp.R

```diff
@@ -0,0 +1,89 @@
+# symComp.R
+
+#'
+#' \code{symComp} Return a list of domains that the genes in two systems
+#'    overlap along with genes that mapped to the same domains
+#'
+#' @param system1 The name of the system in interest. Is compared with system2
+#'     in this function.
+#' @param system2 The name of the system in interest. Is compared with system1
+#'     in this function.
+#' @return A list of overlapping Pfam domains along with genes that mapped to them.
+#'     Also, a horizonal bar graph is generated to summarize the count of genes
+#'     mapped to each overlapping domains.
+#' @author {Tina Lee} (aut)
+#' @examples
+#' # Picking sample system1 and system2 to get intersect domains of the
+#' # two systems
+#' # Call the symComp helper function to generate list
+#' result <- symComp("PHALY", "SLIGR")
+#'
+#' @export
+
+symComp <- function(system1, system2) {
```

Comment:
Perhaps consider naming the function "sysComp" to make it more clear that it's comparing two systems?